var/spack/repos/jcsda-emc-bundles/packages/jedi-tools-env/package.py

```diff
@@ -11,16 +11,24 @@
 class JediToolsEnv(BundlePackage):
     """Development environment for jedi-tools"""
 
+    # DH* TODO CHANGE FROM INTERNAL TO PUBLIC
     homepage = "https://github.com/JCSDA-internal/jedi-tools"
     git      = "https://github.com/JCSDA-internal/jedi-tools.git"
 
     maintainers = ['climbfuji', 'rhoneyager']
 
     version('main', branch='main')
 
-    depends_on('py-click', type='run')
-    depends_on('py-pandas', type='run')
-    depends_on('py-pygithub', type='run')
-    depends_on('py-openpyxl', type='run')
+    variant('latex',
+            default=False,
+            description='Enable building LaTeX documentation with Sphinx')
+
+    depends_on('py-click',                               type='run')
+    depends_on('py-openpyxl',                            type='run')
+    depends_on('py-pandas',                              type='run')
+    depends_on('py-pygithub',                            type='run')
+    depends_on('py-sphinx',                              type='run')
+    depends_on('py-sphinxcontrib-bibtex', when='+latex', type='run')
```

Comment:
Missing dependencies for building LaTeX/PDF version of sphinx doc in spack-stack